src/styles/app/05-main-area.css

```
@@ -725,17 +727,80 @@
+:root[data-translucent-sidebar="false"] .notesInfoSidebarHost:not(:empty) {
+	background: var(--bg-sidebar);
+	border-left-color: var(--border-light);
+}
+
 .mainTabsBarWrap {
 	flex-shrink: 0;
 }
 
 .mainTabsBar {
+	--main-tabs-right-reserve: 78px;
 	height: 40px;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)